pkg/update/root.go

```diff
@@ -68,8 +90,10 @@ func (c *RootCommand) Exec(in io.Reader, out io.Writer) error {
 	}
 
 	if err := os.Rename(latestPath, currentPath); err != nil {
-		progress.Fail()
-		return fmt.Errorf("error moving latest binary in place: %w", err)
+		if err := copyFile(latestPath, currentPath); err != nil {
```

Comment:
We already have a more thorough implementation of copy file in the `common` package. https://github.com/fastly/cli/blob/master/pkg/common/file.go#L21 so it would be good to use that instead here.